vendor/index.yaml

```diff
@@ -158,13 +158,14 @@ vendors:
   - id: decentlab
     name: Decentlab GmbH
     vendorID: 109
-    website: https://www.decentlab.com/
+    website: https://www.decentlab.com
```

Comment:
Can you rebase please? There shouldn't be a diff in this file other than Milesight if you make changes there.